README.md

```diff
@@ -64,7 +64,7 @@ export class ExampleView extends Component {
     this.webview.messagesChannel.on('custom-event-from-webview', eventData => console.log(eventData));
   }
 
-  sendMessageToWebView = () => {
+  sendMessageToWebView() {
```

Comment:
correct me if I'm wrong, but this method is used as `onPress` handler of `TouchableHighlight`. With this change context will be lost and `this.webview.sendJSON` will cause `Cannot read property webview of undefined`

Reply:
Your are right, you would lose the binding if you don't bind on constructor.